aips/passkeys.md

```diff
@@ -0,0 +1,504 @@
+
+## Testing (Optional)
+
+Testing can be found in the reference implementation provided in the PR link above
+
+## Risks and Drawbacks
+
+### Backup Eligibility 
+
+As mentioned in AIP 61:
+
+> **Passkeys are not always backed up to the cloud** on some platforms (e.g., Microsoft Windows). Furthermore, passkeys introduce a **cross-device problem**: if a user creates a blockchain account on their Apple phone, their passkey secret key is backed up to iCloud, which will not be accessible from that user’s other Android device, Linux device or Windows device, since support for cross-platform backup of passkeys is not yet implemented.
+
+For this reason, it is important to check the `backupEligible` and `backupState` flags from the `authenticatorData` in the registration response to ensure that the passkey has been backed up properly.
+
+#### Operating Systems
+
+See the [operating system reference](https://passkeys.dev/docs/reference/) to better understand what operating systems support passkeys and their limitations. Most notably, passkeys registered on Windows devices will NOT be backup-able as of February 2023. If the user would like the passkey to be backed up, it is recommended that they use a device that allows for multi-device credentials and supports passkey backup.
+
+#### Browsers
+
+See this [device support matrix](https://passkeys.dev/device-support/) for more info on which browsers and operating systems enable backup-able passkeys.
+
+### Wallet liveness
+
+An important **liveness consideration** is that the relying party that is tied to your passkey must be available in order for users to access their accounts. 
+
+Several supported browsers, like Google Chrome, that implement the Client to Authenticator Protocol ([CTAP](https://fidoalliance.org/specs/fido-v2.0-ps-20190130/fido-client-to-authenticator-protocol-v2.0-ps-20190130.html)), which regulates the communication protocol between a roaming authenticator and the browser, check to see if the relying part is available before allowing the user to register or make an assertion with the passkey.
+
+In other words, if the relying party is unavailable or returns an error like a 404, the user will not be able to use the passkey associated with that relying party until the relying party comes back online.
+
+To learn more about how chromium handles assertion and registration requests, see [`webauthn_handler.h`](https://github.com/chromium/chromium/blob/95bb60bf7fd3d18f469f050b60663b3dbdfa0402/content/browser/devtools/protocol/webauthn_handler.h#L19)
+
+### Compromised cloud provider
+
+If your passkey is a multi-device credential, backed up to iCloud or Google Password Manager, and the associated cloud account is compromised, so is your passkey account.
+
+Note this only affects multi-device credentials, not hardware-bound credentials like Yubikeys as those are not backed up.
+
```

Comment:
You did not talk about two important risks you mentioned before:
 - Losing the PK means losing the account (and mitigations via ECDSA pubkey recovery)
 - "If the user creates another credential with the same user handle as an existing credential on that authenticator, it will **OVERWRITE** the existing credential"
 
 Both of these need to be addressed.